spec.bs

```diff
@@ -1699,6 +1704,45 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>notifyEvent(|event|)</dfn> method steps are:
+
+  1. [=Assert=]: [=this=]'s {{Document}} is [=fully active=].
```

Comment:
Why is this assert true? Can't you call this API on a detached document?

Reply:
Not being "fully active" is a superset of being detached. Detached documents are all "not fully active". Chromium handles this in https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/html/fenced_frame/fence.cc;l=425-430;drc=599cc06027321ac29b8b518dfb8f4de5f7449254.

You can get and interact with a detached frame or document like so:
```js
const iframe = document.createElement('iframe');
document.body.append(iframe);
const innerFence = iframe.contentWindow.fence;
iframe.remove();
innerFence.notifyEvent(); // Chromium will gracefully early-return. The spec should too.
```

Reply:
I changed the assert here to an early-return.